src/main/java/jalau/at18/kingoftokyo/controller/WhoStartRollDiceController.java

```diff
@@ -0,0 +1,36 @@
+package jalau.at18.kingoftokyo.controller;
+
+import jalau.at18.kingoftokyo.DiceFace;
+import jalau.at18.kingoftokyo.DiceRoller;
+import jalau.at18.kingoftokyo.view.rolldicesection.DiceFaceLabel;
+import jalau.at18.kingoftokyo.view.rolldicesection.RollDiceSectionUI;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WhoStartRollDiceController {
+    private static final int QUANTITY_DICES = 6;
+    private RollDiceSectionUI rollDiceSectionUI;
+
+    private List<DiceFace> listRandomDiceFace;
+
+    public WhoStartRollDiceController(RollDiceSectionUI rollDiceSectionUI) {
+        this.rollDiceSectionUI = rollDiceSectionUI;
+        rollDiceSectionUI.getRollerDiceButton().addActionListener(e -> rollDices());
```

Comment:
Please, Explain me, what is the function of "e ->". I never use it

Reply:
This is a Lambda where for this case we can call the listener.